### PR TITLE
PN-439: Rename "score", "phenScore", "genScore" to "minScore", "minPhenScore", "minGenScore" to make it more clear that this is the minimum value cutoff

### PR DESCRIPTION
**Pull request by veronikaslc**
Done

### CODE REVIEW

File: matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js [hunk 1]
```diff
@@ -539,9 +539,9 @@ var PhenoTips = (function (PhenoTips) {
         var phenScore = (Math.ceil(this._phenScoreSlider.value*20)/20).toFixed(2);
         var genScore = (Math.ceil(this._genScoreSlider.value*20)/20).toFixed(2);
 
-        var options = { 'score'        : score,
-                        'phenScore'    : phenScore,
-                        'genScore'     : genScore};
+        var options = { 'minScore'              : score,
+                        'minPhenotipicScore'    : phenScore,
+                        'minGenotipicScore'     : genScore};
```
Comment (marta-) [inline matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js]: The param names were misspelled in the spec. Please rename:
* `minPhenotipicScore` -> `minPhenotypicScore` (or simply `minPhenScore`)
* `minGenotipicScore` -> `minGenotypicScore` (or simply `minGenScore`)

File: matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/internal/DefaultMatchingNotificationResource.java [hunk 2]
```diff
@@ -343,6 +343,9 @@ private Response getMatchesResponse(@Nullable final double score, @Nullable fina
             matchesJson = buildMatchesJSONArray(matches);
             matchesJson.put(REQ_NO, reqNo);
         }
+        matchesJson.put("minScore", score);
+        matchesJson.put("minPhenotipicScore", phenScore);
+        matchesJson.put("minGenotipicScore", genScore);
```
Comment (marta-) [inline matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/internal/DefaultMatchingNotificationResource.java]: Please add `fromDate` and `toDate` parameters to the response if their values are meaningful.

Comment (marta-) [reply matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/internal/DefaultMatchingNotificationResource.java]: Now that I see the returned JSON I would like to suggest some structure improvements:

```
{
  reqNo : <number>,
  parameters : {
    minScore: ...,
    minPhenScore : ...,
    minGenScore : ...,
    fromDate : ...,
    toDate: ...
  },
  dateGenerated: 'yyyy-mm-ddTHH:MM:ssZ',
  results : [...],
  returnedCount : ...
}
```

Basically the changes are:
* Group all parameters provided by the user under `parameters`
* include a top-level field that specifies when the data was generated

**Comment by veronikaslc** [comment]
commit amended wrt latest changes

**Pull request merged**